hack/pr-deploy.yaml

```diff
@@ -170,7 +170,7 @@ objects:
           - /git-sync
           args:
           - --repo=https://github.com/openshift/release.git
-          - --branch=master
+          - --branch=${RELEASE_BRANCH}
```

Comment:
I don't think this will work. Will `oc` command resolve that var from the env vars?